__fixtures__/base/disallowed.js

```diff
@@ -0,0 +1,41 @@
+const cp = require( 'child_process' );
+
+const lol = 'rm -rf all_your_files';
+cp.exec( lol );
```

Comment:
This line isn't getting marked as a warning or error in my IDE (here or in the cil/disallowed fixture.  Should it be?

Reply:
Did you `npm i` to pick up the new security plugin?

Reply:
Oh, whoops. I did when I started looking at this branch, but not after you added the security plugin. Don't mind me!